overlay-ot.conf

```diff
@@ -0,0 +1,49 @@
+# Hack: Disable BT on devices which support both for now
```

Comment:
Can we just change the application Kconfig to say:


```
config BT
	default y if !NET_L2_OPENTHREAD
```

?

Reply:
This is actually an open item for Nordic to fix. 
See zephyr/boards/arm/nrf52840_pca10056/nrf52840_pca10056_defconfig line 22
CONFIG_BT=y

Reply:
Got it, thanks.